theano/tensor/tests/test_opt.py

```diff
@@ -4284,8 +4282,8 @@ def test_local_log_erfc(self):
             0].op.scalar_op.fgraph.apply_nodes) == 22, len(f.maker.fgraph.toposort()[0].fgraph.toposort()[0].op.scalar_op.fgraph.apply_nodes)
         # TODO: fix this problem
         if theano.config.floatX == "float32" and theano.config.mode in ["DebugMode", "DEBUG_MODE"]:
-            raise KnownFailureTest(
-                "the python code upcast somewhere internally some value of float32 to python float for part of its computation. That make that the c and python code don't generate the same value. You can ignore this error.")
+            raise SkipTest('The python code upcast somewhere internally some value of float32 to python float for part of its computation.'
+                'That make that the c and python code dont generate the same value. You can ignore this error.')
```

Comment:
`do not` or `don\'t` or use double quotes, but you should not skip apostrophes like that.